webpack.config.js

```diff
@@ -1,29 +1,28 @@
 // import HtmlWebpackPlugin from 'html-webpack-plugin';
```

Comment:
- [ ] OPTIONAL
Webpack is not configured properly and hence your application is not working as intended. You can update your webpack to look something similar to this:

```javascript
const path = require('path');
const HtmlWebpackPlugin = require('html-webpack-plugin');

module.exports = {
  mode: 'development',
  entry: './src/index.js',
  output: {
    filename: 'main.js',
    path: path.resolve(__dirname, 'dist'),
  },
  module: {
    rules: [{
      test: /\.css$/i,
      use: ['style-loader', 'css-loader'],
    }],
  },
  devServer: {
    contentBase: './dist',
  },

  plugins: [
    new HtmlWebpackPlugin({
      template: './src/index.html',
    }),
  ],
};
```